manifests/install.pp

```diff
@@ -4,31 +4,29 @@
 
   $modulestream_nodejs = $katello_devel::modulestream_nodejs
```

Comment:
You can consider removing this variable and in the `ensure` use `$katello_devel::modulestream_nodejs`.